### ISSUE TITLE
[BUG] Submit button blocked by iOS keyboard for Illinois Assistant feedback

### ISSUE THREAD

**Issue by cdukeUX** [rokwire/illinois-app#4267]
The iOS keyboard blocks the "Submit" button when submitting text feedback to the Illinois Assistant.

*See screen recording below*

1. Open the app in production on iOS
2. Tap on the Illinois Assistant tab
3. Enter a question in the query field and send
4. View the response and tap on the thumbs down icon
5. See the field labeled "Thank you for providing feedback! Can you briefly explain the issue with the response?"
6. Enter text in the field
7. Attempt to tap the "Submit" button

There is no way to hide the iOS keyboard in order to tap the Submit button. On Android, a small arrow can be used to minimize the keyboard to fully view and access the "Submit" button."

**Screen Recording** [Illinois app 6.0.27, Apple iOS 17.5.1]
The link provided includes a screen recording demonstrating steps 1-7: https://drive.google.com/open?id=1ds9NC4aVibZnSLbOu1NVfgSxGPEVwvn1&usp=drive_fs

CC: @vburgett

**Comment by TodorBachvarov** [comment]
This should be fixed in version 6.0.30

**Comment by cdukeUX** [comment]
Thanks! @vburgett was able to do a preliminary test on 6.0.30 with her iOS build and the scrollable size seems to be an improvement from the issue I was encountering. More testing will be needed when others (myself included) have access to test on iOS after release.

To further improve the experience of using the feedback box, please make the following changes (see attached screenshot). 
1. Add an "X" with circle icon to close out the feedback box
2. Shorten the text from "Thank you for providing feedback! (line break) **Can you briefly explain the issue(s) with the response?** [bold]" to "Thanks for your feedback! **Please provide additional information on the issue(s).** [bold]"
3. Decrease the line spacing to decrease the overall vertical spacing of the box (16pt font should have no more than 18pt line spacing; 14pt font should have no more than 16pt line spacing)
4. Decrease the vertical height of the comment field by half (enough for just two lines of text)

<img width="1876" alt="5-7 Update Negative Feedback Comment Field" src="https://github.com/user-attachments/assets/c7f175e8-c783-4c57-ba0a-72a32c2bff94">

**Comment by cdukeUX** [comment]
(By shortening the height of the feedback box and bringing the Submit button up higher, we will hopefully avoid several UX issues caused by the lack of a iOS keyboard minimizer.)

**Comment by cdukeUX** [comment]
@TodorBachvarov

**Comment by cdukeUX** [comment]
Closing this out as the button is no longer blocked. Will resubmit the UI changes as part of a new ticket!
